app/scripts/background.js

```diff
@@ -570,6 +563,9 @@ async function initialize(backup) {
   // In MV3, the Service Worker sees `navigator.webdriver` as `undefined`, so this will trigger from
   // an Offscreen Document message instead. Because it's a singleton class, it's safe to start multiple times.
   if (process.env.IN_TEST && window.navigator?.webdriver) {
+    const { getSocketBackgroundToMocha } =
+      // eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires, node/global-require
```

Comment:
Nit: Thoughts on adding a comment to explain why we're doing this? e.g.

```suggestion
      // Use `require` to make it easier to include this test code from the build
      // eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires, node/global-require
```

Reply:
```suggestion
       // Use `require` to make it easier to exclude this test code from the Browserify build.
      // eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires, node/global-require
```
Sure something like this?

Reply:
@davidmurdoch Not possible everywhere, but used it where possible: https://github.com/MetaMask/metamask-extension/pull/34690/commits/aa4ec87110082102ecb56cecb292e8cbdbed8e11